ds_caselaw_editor_ui/templates/judgment/unhold-success.html

```diff
@@ -0,0 +1,13 @@
+{% extends "layouts/judgment_with_sidebar.html" %}
+{% load waffle_tags %}
+{% block judgment_content %}
+  {% load i18n %}
+  <h2>
+    {% translate 'judgment.hold.unhold_success_title' %}
+    <br>
+    {{ context.judgment.name }}
+  </h2>
+
+  <p>{% translate "judgment.unhold.success" %}</p>
```

Comment:
```suggestion
  <p>{% translate "judgment.hold.unhold_success_message" %}</p>
```
Again super nitpicky, but I think this better conveys the intent of this string.